db/migrate/20220125172503_remove_inventory_from_user.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveInventoryFromUser < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :inventory, :json
```

Comment:
You don't need another migration for this if `add_column` is in the same branch. All you had to do was to rollback and get rid of `db/migrate/20220123190531_add_inventory_to_users.rb`
Let's remove both files

Reply:
Migrations have been removed